robot_modeling/DQ_SerialManipulatorDH.m

```diff
@@ -160,13 +116,15 @@
             
             % The optimized standard dh2dq calculation
```

Comment:
I did not add that comment. I think that the comment was added in the [first commit](https://github.com/dqrobotics/matlab/commit/fafd4dad68804ffd1f0c3ab99ed315bf35adfda6) of the DQ_SerialManipulatorDH.m (in the master branch) by @mmmarinho.  
@mmmarinho, assuming that I am not wrong, would you answer the question about the "optimized calculation"?

Reply:
@bvadorno @juanjqo 

These comments were in the last release version (20.04).
https://github.com/dqrobotics/matlab/blob/f11f0de06784d04042b8713010373d08fb86ccf8/robot_modeling/DQ_SerialManipulatorDH.m#L183

All comments might be removed without any loss, I believe. Especially if they are causing confusion.

Anyways, the "optimized calculation" is the [simplified](https://www.mathworks.com/help/symbolic/simplify.html) form of the `dh2dq` transformation that can be used instead of multiplying the four dual quaternions in sequence, e.g. `h1*h2*h3*h4`. The half angles are a small additional detail.